src/main.cpp

```diff
@@ -2596,7 +2596,7 @@ void static UpdateTip(CBlockIndex *pindexNew, const CChainParams& chainParams) {
             ThresholdState state = checker.GetStateFor(pindex, chainParams.GetConsensus(), warningcache[bit]);
             if (state == THRESHOLD_ACTIVE || state == THRESHOLD_LOCKED_IN) {
                 if (state == THRESHOLD_ACTIVE) {
-                    strMiscWarning = strprintf(_("Warning: unknown new rules activated (versionbit %i)"), bit);
+                    strMiscWarning = strprintf("Warning: unknown new rules activated (versionbit %i)", bit);
```

Comment:
Well the point is that it should not appear translated on the API (RPC).
There is currently no way to set a different message here for the GUI and for the RPC.
There should be one, but the point here was to do a quick change. I don't think it's worth spending a lot of time here now, so I'm going to close this.